packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -185,17 +192,20 @@ const Alert = ({
     setIsVisible(false);
   };
   const closeButton = isDismissable ? (
-    <IconButton
-      accessibilityLabel="Dismiss alert"
-      onClick={onClickDismiss}
-      contrast={contrast}
-      size="large"
-      icon={CloseIcon}
-    />
+    <CloseButtonWrapper>
+      <IconButton
```

Comment:
Did we decide on a `hitSlop`? The icon button is pretty difficult to click 😅

Reply:
This is pending design inputs tracking https://github.com/razorpay/blade/issues/639 /cc @saurav12 😬